clients/go/cmd/zbctl/internal/commands/output.go

```diff
@@ -39,38 +38,21 @@ func addOutputFlag(c *cobra.Command) {
 	)
 }
 
-func printHumanAndJSON(p Printable) error {
-	var responseAsString string
+func printOutput(p Printable) error {
```

Comment:
This is only used for "good" output right? Here I mean, uh, not errors basically - I imagine errors are logged to STDERR?

Reply:
Yes, errors are handled through Cobra